tests/frontend/test_ops_gate.py

```diff
@@ -53,7 +53,7 @@ def Q(self):
     def G(self, gate):
         """Initialize each gate"""
         if gate in ops.zero_args_gates:
-            return gate
+            return gate()
```

Comment:
What caused the change here?

Reply:
`Fouriergate` (with shortcut variable `Fourier = Fouriergate()`) is the only zero argument gate.

In this loop, it used to receive `Fourier`, which was pre-initialized. Now it receives `Fouriergate` instead, which need to be initialized.